base/sparse/sparsevector.jl

```diff
@@ -560,6 +560,55 @@ function getindex{Tv}(A::SparseMatrixCSC{Tv}, I::AbstractVector)
     SparseVector(n, rowvalB, nzvalB)
 end
 
+function find{Tv,Ti}(x::SparseVector{Tv,Ti})
+    numnz = nnz(x)
+    I = Array(Ti, numnz)
+
+    nzind = x.nzind
+    nzval = x.nzval
+
+    count = 1
+    @inbounds for i = 1 : numnz
+        if nzval[i] != 0
+            I[count] = nzind[i]
+            count += 1
+        end
+    end
+
+    count -= 1
+    if numnz != count
+        deleteat!(I, (count+1):numnz)
```

Comment:
I don't think so. It seems to all dispatch to the same stuff eventually in the runtime. Do we have evidence otherwise?